lte/gateway/python/magma/subscriberdb/main.py

```diff
@@ -136,5 +140,15 @@ def _get_s6a_manager(service, processor):
     )
 
 
+def _randomize_sync_interval(interval: int) -> int:
+    """_randomize_sync_interval increases sync interval to ameliorate
+    thundering herd at Orc8r."""
+    h = hashlib.md5()
+    h.update(bytes(snowflake.snowflake(), 'utf8'))  # digest of hwid
+    multiplier = (hash(h.hexdigest()) % 100) / 100  # to interval [0, 1]
+    delta = multiplier * (interval / 5)  # up to 1/5 of target interval
```

Comment:
looks a bit convoluted, why not use random?

Reply:
I agree, using random library might be more straightforward.
```
random.seed(snowflake.snowflake())
random.uniform(interval, interval + (interval/5)) 
```

Reply:
To make the delta deterministic. Discussed offline with Karthik, will add a comment to make the intent clearer